docs/usage.md

```diff
@@ -8,7 +8,23 @@ The software has been tested with MATLAB R2021B on an Ubuntu 20.04 LTS OS. The r
 - MATLAB Curve Fitting Toolbox
 
 ### Easier way to install dependencies
-To be updated.
+Instead of manually installing the iDynTree MATLAB bindings, we provide a script that installs the bindings using a `Mambaforge` installer without modifying anything in the PC.
+
+One can install these bindings by launching the script `install_idyntree_bindings` on the MATLAB console while being in the `scripts` folder. This process must install the bindings in `paper_ramadoss-2022-ral-humanoid-base-estimation/deps/idyntree-matlab/mex`.
+
+In order to install the bindings in a custom location, one can run
+
+``` matlab
+install_idyntree_bindings('installPrefix', '<install_path>')
+```
+
+One must ensure not to provide any path location with a space character in it, eg. `/deps/hello world` is not recommended.
```

Comment:
I would avoid to put this information here, it seems to add complexity that is not needed. If necessary, we can put this info in the comments of the `install_idyntree_bindings` function, so that expert users if they want know where to look for it.

Reply:
Addressed in https://github.com/ami-iit/paper_ramadoss-2022-ral-humanoid-base-estimation/commit/2295787f336f3825bc6366453dbd792b52198801